### DISCUSSION TITLE
Google Calendar link

### DISCUSSION THREAD

**Discussion by Joostvab** [liedekef/events-made-easy#115] [General]
I'd like to add a google calendar link in the e-mail but when I do this the link is transformed to a "send in blue" link. The email host.

The used link:
<a target="_blank" href="http://www.google.com/calendar/event?action=TEMPLATE
&text=#URL_NAME
&dates=#_{Ymd}T#H#_{i}00/#@_{Ymd}T#@H#@_{i}00
&sprop=#URL_EVENTPAGEURL
&location=#URL_ADDRESS,+#URL_TOWN
&details=#URL_NOTES"
>Add to Google Calendar</a>

All in  on one line:

If i put the code in the description page of the event the #field are not filled.

My question is how can this be done?

Like to hear from you

Joost

**Comment by liedekef** [comment]
The event description is part of the event, like is the date/time/... how you show all that is decided in the single event format. So change that to your liking, either in the EME settings or via a template and using that template in [eme_events].
Next to that: EME doesn't change links to send-in-blue, so that is probably done by your sending email server.
